Gruntfile.js

```diff
@@ -259,7 +308,49 @@ module.exports = function (grunt) {
         grunt.task.run(tasks);
     });
 
-    grunt.registerTask("buildStylus", ["clean:stylus", "stylus"]);
+    grunt.registerMultiTask("distributions", "Enables a project to split its files into a set of modules. A module's information is stored in a json file containing a name for the module, the files it contains, and other modules it depends on. The module files can then be accumulated into various configurations of included and excluded modules, which can be fed into other plugins (e.g. grunt-contrib-concat) for packaging.", function () {
+        // Merge task-specific and/or target-specific options with these defaults.
+        var options = this.options({
+            name: this.target,
+            source: true,
+            target: "all",
+            compress: false
+        });
+
+        if (options.exclude || options.include) {
+            options.target = "custom";
+        }
+
+        setBuildSettings(options);
+
+        var concatTask = options.compress ? "uglify:" : "concat:";
+        var tasks = [
+            "clean:build",
```

Comment:
The third option would be nice, if it didn't seem to involve too much work - otherwise we could just leave things are they are